packages/cozy-konnector-libs/src/libs/BaseKonnector.js

```diff
@@ -163,6 +164,40 @@ class BaseKonnector {
     }
   }
 
+  /**
+   * Read an eventual payload from COZY_PAYLOAD env var, wether it is a JSON string or a reference
+   * to a file containing a JSON string
+   *
+   * @returns Promise<{Object|null}> result of JSON.parse from the JSON string or null if no payload
+   */
+  async readPayload() {
+    const cozyPayload = process.env.COZY_PAYLOAD
+
+    if (cozyPayload == null) {
+      return null
+    }
+
+    const isFileReference = get(cozyPayload, '[0]') === '@'
+
+    if (isFileReference) {
+      const filePath = cozyPayload.substr(1)
+      try {
+        const fileContent = await fs.readFile(filePath)
```

Comment:
What does it matter if the file size is 10 or 15MB? Won't it blow up the server's RAM? Do we have an idea of the max size of the BI's payload?

Reply:
@doubleface ? Should we use chunk or something ?